<ui version="4.0" >
 <class>userEditWindow</class>
 <widget class="QWidget" name="userEditWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Identyfikator Tlen.pl</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="le_jid" >
        <property name="toolTip" >
         <string>Identyfikator użytkownika, bez '@tlen.pl'</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Nazwa</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="le_nick" >
        <property name="toolTip" >
         <string>Nazwa użytkownika (imię, pseudonim)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Grupa</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QComboBox" name="cb_group" >
        <property name="toolTip" >
         <string>Grupa</string>
        </property>
        <property name="editable" >
         <bool>true</bool>
        </property>
        <property name="frame" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
